README.md

```diff
@@ -1,146 +1,59 @@
-liboqs - master branch
+[AppVeyor](https://ci.appveyor.com/project/dstebila/liboqs): ![Build status image](https://ci.appveyor.com/api/projects/status/9d2ts78x88r8wnii/branch/master?svg=true), [CircleCI](https://circleci.com/gh/open-quantum-safe/liboqs/tree/master): ![Build status image](https://circleci.com/gh/open-quantum-safe/liboqs/tree/master.svg?style=svg), [Travis CI](https://travis-ci.org/open-quantum-safe/liboqs): ![Build status image](https://travis-ci.org/open-quantum-safe/liboqs.svg?branch=master)
+
+liboqs
 ======================
 
-**liboqs** is a C library for quantum-resistant cryptographic algorithms.  This branch of liboqs focuses on selected key encapsulations and signature algorithms.
+liboqs is an open source C library for quantum-safe cryptographic algorithms.
 
-Overview
---------
+- [Overview](#overview)
+- [Status](#status)
+- [Quickstart](#quickstart)
+- [Docs](#docs)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgements](#acknowledgements)
 
-The **Open Quantum Safe (OQS) project** has the goal of developing and prototyping quantum-resistant cryptography.
+## Overview
 
-**liboqs** is an open source C library for quantum-resistant cryptographic algorithms.  liboqs provides:
+liboqs provides:
 
-- a common API for post-quantum key encapsulation mechanisms and digital signature schemes
-- a collection of open source implementations of post-quantum cryptography algorithms
+- a collection of open source implementations of quantum-safe key encapsulation mechanism (KEM) and digital signature algorithms; the full list can be found [here](docs/algorithms)
+- a common API for these algorithms
 - a test harness and benchmarking routines
 
-The OQS project also provides prototype integrations into application-level protocols to enable testing of quantum-resistant cryptography.
-
-More information on OQS can be found on our website: [https://openquantumsafe.org/](https://openquantumsafe.org/).
-
-master branch
--------------
-
-This branch of liboqs aims to selectively incorporate allegedly quantum-resistant key encapsulation mechanisms and signature schemes, for the purposes of integration into a common API for liboqs-reliant applications.
-
-Implementations on this branch must meet certain acceptance criteria as indicated below.
-
-Algorithms currently included in liboqs master branch include:
```

Comment:
I'd like to keep the list of algorithms in the main README, even though this also appears in docs/algorithms and in the source code.